src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_fields_table/index.js

```diff
@@ -0,0 +1 @@
+export default from './scripted_fields_table';
```

Comment:
Should this be `import './scripted_fields_table';` instead, since nothing is being exported by that file?